test/unit-test/FreeRTOS_TCP_Transmission/FreeRTOS_TCP_Transmission_utest.c

```diff
@@ -407,21 +451,30 @@ void test_prvTCPReturnPacket_Null_Socket( void )
     usGenerateChecksum_ExpectAnyArgsAndReturn( 0x1111 );
     usGenerateProtocolChecksum_ExpectAnyArgsAndReturn( 0x2222 );
     eARPGetCacheEntry_ExpectAnyArgsAndReturn( eARPCacheHit );
-    xNetworkInterfaceOutput_ExpectAnyArgsAndReturn( pdTRUE );
+    //xNetworkInterfaceOutput_ExpectAnyArgsAndReturn( pdTRUE );
```

Comment:
Please remove this if it's a debug leftovers.

Reply:
sure, will update that

Reply:
fixed all